routers/web/web.go

```diff
@@ -236,6 +236,10 @@ func RegisterRoutes(m *web.Route) {
 	if setting.Federation.Enabled {
 		m.Get("/.well-known/nodeinfo", NodeInfoLinks)
 	}
+	m.Get("/.well-known/change-password", func(w http.ResponseWriter, req *http.Request) {
```

Comment:
We might want to switch to a `m.Group("/.well-known", func() {`

Reply:
Ok, I did it in a 2nd commit (for easier review), but I can squash them if people prefer.